File: examples/src/Presentations/Conversion/RenderingNotesWhileConvertingToHTML.cs
using Aspose.slides.Export


/*
This project uses Automatic Package Restore feature of NuGet to resolve Aspose.Slides for .NET API reference 
when the project is build. Please check https:#docs.nuget.org/consume/nuget-faq for more information. 
If you do not wish to use NuGet, you can manually download Aspose.Slides for .NET API from http:#www.aspose.com/downloads, 
install it and then add its reference to this project. For any issues, questions or suggestions 
please feel free to contact us using http:#www.aspose.com/community/forums/default.aspx
*/

namespace Aspose.slides.Examples.CSharp.Conversion
{
    public class RenderingNotesWhileConvertingToHTML
    {
        public static void Run()
        {
            #ExStart:RenderingNotesWhileConvertingToHTML
            # The path to the documents directory.
            dataDir = RunExamples.GetDataDir_Conversion()

            using (Presentation pres = new Presentation(dataDir + "Presentation.pptx"))
            {
                HtmlOptions opt = new HtmlOptions()

                INotesCommentsLayoutingOptions options = opt.NotesCommentsLayouting
                options.NotesPosition = NotesPositions.BottomFull

                # Saving notes pages
                pres.save(dataDir + "Output.html", SaveFormat.Html, opt)
            }
            #ExEnd:RenderingNotesWhileConvertingToHTML
        }
    }
}

File: examples/src/presentations/conversion/rendering_notes_while_converting_to_html.py
import aspose.slides as slides


def convert_to_html_with_notes(global_opts):
    with slides.Presentation(global_opts.data_dir + "welcome-to-powerpoint.pptx") as pres:
        html_options = slides.export.HtmlOptions()

        slides_layout_options = slides.export.NotesCommentsLayoutingOptions()
        slides_layout_options.notes_position = slides.export.NotesPositions.BOTTOM_FULL
        html_options.slides_layout_options = slides_layout_options

        # Saving notes pages
        pres.save(global_opts.out_dir + "convert_to_html_with_notes_out.html", slides.export.SaveFormat.HTML,
                  html_options)
